src/components/TokenClaimDialog.tsx

```diff
@@ -72,6 +93,7 @@ export default function TokenClaimDialog({
   const formattedTotalRetroDrop = commify(
     formatBNToString(rewardBalances.retroactiveTotal, 18, 2),
   )
+
   const [allPoolsWithRewards, poolsWithUserRewards] = useMemo(() => {
```

Comment:
adapt this to work with gauges, or delete it. We do like that it's sorted by reward balance though.